README.md

```diff
@@ -104,23 +113,68 @@ $ docker-compose ps
 $ docker-compose logs -f
 ```
 
-**Note**: 
-There is a time drift issue on Docker for Mac, to solve it try running [Fixing Time drift issue on Docker for Mac](https://blog.shameerc.com/2017/03/quick-tip-fixing-time-drift-issue-on-docker-for-mac):
+## Running on DappNode
+
+Building and running the TrueBlocks docker image is taken care of by the dAppNode package manager. If you are running a dAppNode, you can simply install the package from the dAppNode package manager.
+
+Note: your DappNode doesn't run Parity with `--tracing=on` by default. Set `EXTRA_OPTS=--tracing on` in the dappnode admin panel ([image](https://user-images.githubusercontent.com/21328788/52904014-04ab6c00-3226-11e9-8c47-747a42b22169.png)) or setting it in DNP_ETHCHAIN's ethchain.dnp.dappnode.eth.env file (see [Why do we need --tracing enabled?](#why-do-we-need-tracing-enabled))
+
+## FAQ
+
+### I'm running geth, do I need to run Parity instead?
+Yes - Parity delivers the necessary articulated traces so that TrueBlocks can build its address index. We don't yet support Geth.
+
+### What is my RPC endpoint?
+
+That depends on how you've configured your parity node. some endpoints that we use are http://172.17.0.1:8545 (when running node on linux host), http://docker.for.mac.localhost:8545 (when running node on mac host), and http://my.ethchain.public.dappnode.eth:8545 (when running node on dappnode). http://localhost:8545 is a common guess, but docker has its own networking paradigm; don't be surprised if this RPC endpoint choice doesn't yield results.
+
+Parity's default http RPC port is 8545, but your node could be configured differently.
+
+### Why does it take so long to build the index?
+TrueBlocks has to manually process every single block in the history of the Ethereum chain. Additionally, it has to descend into every transaction trace. Often, traces are deeply layered (traces of traces of traces of ... traces), and this takes time to 1) fetch from your node's RPC and 2) extract addresses.
```

Comment:
When we say "every single block" people will say "I can do that from Infura really quickly" (because they sort of can. Maybe we say something like "When you sync your Ethereum node with --tracing on, the node takes more than three weeks to sync. This is because the node is replaying every transaction and every state change that every happened. TrueBlocks must do the same as it decends into every trace which are not stored by the node but must be re-generated..."  Something like that. Blame it on the nature of the beast -- which is where the blame lies after all.

Reply:
I'm going to leave this the way it is in order to get the PR through. That being said, I encourage you to get your better articulation across. Let's also consider creating a separate wiki where more in-depth explaining can be done.